docs/source/general/finetuning.rst

```diff
@@ -0,0 +1,115 @@
+**********
+Finetuning
+**********
+
+Finetuning (or transfer-learning) is the process of tweaking a model trained on a large dataset, to your particular (likely much smaller) dataset. All Flash tasks have a pre-trained backbone that was trained on large datasets such as ImageNet, and that allows to decrease training time significantly. 
+
+.. note:: If you have a huge dataset and prefer to train from scratch, see the training guide.
+
+You can finetune any Flash tasks on your own data in just a 3 simple steps:
+
+1. Load your data and organize it using :ref:`Flash Data Modules <data>`. Note that different tasks have different data modules (The :class:`~flash.vision.ImageClassificationData` for image classification, :class:`~flash.text.TextClassificationData` for text classification, etc.).
+
+2. Pick a model to run from a variety of Flash tasks: :class:`~flash.vision.ImageClassification`, :class:`~flash.text.TextClassifier`, :class:`~flash.tabular.TabularClassifier`, all optimized with the latest best practices.
+
+3. Finetune your model using  :func:`~flash.Trainer.finetune` method. You will need to choose a finetune strategy.
+
+
+Finetune options
+================
+
+Flash supports 4 builts-in Finetuning options:
+
+* `no_freeze`: Don't freeze anything.
+* `freeze`: Freeze the backbone parameters when training starts.
+* `freeze_unfreeze`: Freeze the backbone parameters when training starts and unfreeze the backbone when reaching `unfreeze_epoch`.
+* `unfreeze_milestones`: Freeze the backbone parameters when training starts and unfreeze the end backbone when reaching first milestones and begining when reaching second one.
```

Comment:
can you specify or link what milestone means?

Reply:
Elaborate on the difference between the "backbone" and the "end backbone".

Also elaborate on what is the beggining

Also "when reaching first milestiones" -> "when the first milestone is reached"